.vscode/settings.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
Is there something we can do about hinting to install the Stylable intelligence plugin? Or leave it as a note maybe on the README

Reply:
Dunno, readme for now...